libkbfs/folder_block_manager.go

```diff
@@ -1306,10 +1306,21 @@ func (fbm *folderBlockManager) clearLastQRData() {
 	fbm.lastReclamationTime = time.Time{}
 }
 
-func (fbm *folderBlockManager) doCleanSyncCache() (err error) {
-	defer fbm.cleanSyncCacheGroup.Done()
+func (fbm *folderBlockManager) doCleanDiskCaches() (err error) {
+	defer fbm.cleanDiskCachesGroup.Done()
+	// TODO: determine if these can be parallelized
+	err = fbm.doCleanDiskCache(DiskBlockSyncCache)
+	if err != nil {
+		return err
+	}
+	return fbm.doCleanDiskCache(DiskBlockWorkingSetCache)
+}
+
+func (fbm *folderBlockManager) doCleanDiskCache(cacheType DiskBlockCacheType) (
+	err error) {
 	syncConfig := fbm.config.GetTlfSyncState(fbm.id)
-	if syncConfig.Mode == keybase1.FolderSyncMode_DISABLED {
+	if syncConfig.Mode == keybase1.FolderSyncMode_DISABLED &&
+		cacheType == DiskBlockSyncCache {
```

Comment:
It seems the syncConfig is only used for determining this. Would it make sense to move this check into `doCleanDiskCaches` so if `DiskBlockSyncCache` is `DISABLED` we'd just not call this function?

Reply:
done.